caluma/caluma_core/visibilities.py

```diff
@@ -37,8 +37,18 @@ class BaseVisibility(object):
     ...     @filter_queryset_for(Form)
     ...     def filter_queryset_for_form(self, node, queryset, info):
     ...         return queryset.exclude(slug='protected-form')
+    ...
+    ...     # Do not trigger visibility when looking up case from workitem
+    ...     suppress_visibilities = [
+    ...         "WorkItem.case",
+    ...         "WorkItem.child_case",
```

Comment:
Should we add all available suppressable visibilities here? Or rather in the caluma docs?